calicoctl/README.md

```diff
@@ -25,8 +25,12 @@ If you want to use a package manager:
 
 - [Homebrew] users can use `brew install calicoctl`.
 
+More detailed installation instructions, including how to install `calicoctl` as a
+Kubernetes pod, are available in the [online documentation][detailed install instructions].
+
 [Releases page]: https://github.com/projectcalico/calicoctl/releases
 [Homebrew]: https://brew.sh/
+[detailed install instructions]: https://projectcalico.docs.tigera.io/master/maintenance/clis/calicoctl/install
```

Comment:
I was going based on a few lines above where the master doc is linked to. I don't want to get that mixed up with the intent of this PR, but I figured you'd want to know in case you want to change that. I'm happy to take your suggestion though!

Reply:
Ah yes, that makes sense. I believe that link is probably wrong as well.